test_scripts/Smoke/commonSmoke.lua

```diff
@@ -322,6 +319,27 @@ function commonSmoke.putFile(params, pAppId, self)
   mobileSession:ExpectResponse(cid, { success = true, resultCode = "SUCCESS"})
 end
 
+function commonSmoke.AppType(HMIType)
```

Comment:
@GetmanetsIrina If purpose of this function is to set the HMIType it would be better to rename it to 'SetAppType'

Reply:
@dboltovskyi, updated.